Makefile

```diff
@@ -56,6 +56,7 @@ lint:
 	  --global module \
 	  --global require \
 	  --global self \
+	  --rule 'indent: [off]' \
```

Comment:
It's disabling `key-spacing` in [the other place](https://github.com/sanctuary-js/sanctuary-type-classes/blob/8ac0f396558081d6e2646ccca7c0fc523966e807/index.js#L986). I don't think it'd be appropriate to remove that in the same commit. If you want I can make a new commit. I would prefer keeping it the way it is though.